integration_testing/features/coach/coach-print-reports.feature

```diff
@@ -0,0 +1,102 @@
+
+    When I print or save the report from *Reports > Quizzes > '<quiz1>' > Difficult questions* subtab 
+    Then on the paper/PDF I see a high level <quiz1> overview on top
+      And below I see the table with the same columns and values for all the questions in <quiz1> that at least 2 learners gave incorrect answers to, as in *Reports > Quizzes > '<quiz1>' > Difficult questions* subtab
+
+
+  Scenario: Check print results for Groups' reports
+
+    When I print or save the report from *Reports > Groups* subtab
+    Then on the paper/PDF I see a table with the same columns and values for <group1> and <group2>, as in *Reports > Groups* subtab
+
+    When I print or save the report from *Reports > Groups > '<group1>' > Reports> '<lesson1>'* page
+    Then on the paper/PDF I see a table with the same columns and values for resources in <lesson1> assigned to <group1>, as in *Reports > Groups > '<group1>' > Reports> '<lesson1>'* page
+
+    When I print or save the report from *Reports > Groups > '<group1>' > Reports> '<quiz1>' > Report* subtab
+    Then on the paper/PDF I see a high level <quiz1> overview on top
+      And below I see a table with the same columns and values for all the learners in <group1> that <quiz1> is assigned to, as in *Reports > Groups > '<group1>' > Reports> '<quiz1>' > Report* subtab
+
+    When I print or save the report from *Reports > Groups > '<group1>' > Reports> '<quiz1>' > Difficult questions* subtab
+    Then on the paper/PDF I see a high level <quiz1> overview on top
+      And below I see a table with the same columns and values for all the questions in <quiz1> that at least 2 learners gave incorrect answers to, as in *Reports > Groups > '<group1>' > Reports> '<quiz1>' > Difficult questions* subtab
+
+    When I print or save the report from *Reports > Groups > '<group1>' > Members* subtab
+    Then on the paper/PDF I see a table with the same columns and values for all the learners in <group1>, as in *Reports > Groups > '<group1>' > Members* subtab
+
+  # Reports printed from *Reports > Groups > '<group1>' > Members > '<learnerN>'* pages are the same as *Reports > Learners > '<learnerN>'* pages
+
+  Scenario: Check print results for Learners' reports
+
+    When I print or save the report from *Reports > Learners* subtab
+    Then on the paper/PDF I see a table with the same columns and values for all the learners in the <class>, as in *Reports > Learners* subtab
+
+    When I print or save the report from *Reports > Learners > '<learner1>' > Reports* subtab
+    Then on the paper/PDF I see a high level <learner1> overview on top
+      And below I see table(s) for lesson(s) and quizzes they have been assigned, with the same columns and values as in *Reports > Learners > '<learner1>' > Reports* subtab
+
+  # Reports printed from *Reports > Learners > '<learnerN>' > '<lessonN>'* pages are essentially the same as *Reports > Lessons > '<lessonN>' > Learners > '<learnerN>'* pages
```

Comment:
The best intention was to make it shorter, but it still ended up being more than 100 lines 🤦‍♀ 

Originally I started spelling out both options for each single view, but it would have been way too repetitive:
```
  Scenario: Print or save as PDF the main lessons view
    Given I am on *Coach - '<class>' > Reports > Lessons* subtab
      When I click the *Print report* button
      Then I see the default print dialog for my chosen browser
      When I select my paper printer device
        And I make other adjustments in the printer settings according to my needs # quality, margins, etc.
        And I confirm # could be *Print* or *OK* depending on the browser
      Then I have a paper version of the *'<class>' Lessons* report
        And on the paper I see a table with *Title*, *Progress* and *Recipients* columns for <lesson1> and <lesson2>
        And the values in the table are the same as on *Coach - '<class>' > Reports > Lessons* subtab
      When I select *Save as PDF* # wording might be different depending on the browser
        And I make other adjustments in the PDF settings according to my needs # headings, margins, etc.
        And I confirm # could be *Save* or *OK* depending on the browser
      Then I have a PDF version of the *'<class>' Lessons* report
        And on the PDF file I see a table with *Title*, *Progress* and *Recipients* columns for <lesson1> and <lesson2>
        And the values in the table are the same as on *Coach - '<class>' > Reports > Lessons* subtab

...and so forth...
```

@indirectlylit Does this look in line of what you had in mind? Can I base the CSV export on this, or does it need some restructuring?